release/models/bgp/openconfig-bgp-policy.yang

```diff
@@ -994,7 +1015,21 @@ module openconfig-bgp-policy {
       "Configuration data for referening a extended community-set
       in the set-ext-community action";
 
+    leaf ext-community-set-refs {
```

Comment:
Should this be a leaf-list?

Reply:
Apologies for not double-checking, fixed.

Any YANG tree diff tools should show the type of the node in addition to the path.